### PR TITLE
About page

### PR DESCRIPTION
**Pull request by sonlinux**
fix kartoza/LEDET_BIMS#160

aboutuspage

### Activate home page link

aboutlink

### CODE REVIEW

File: bims/admin.py [hunk 1]
```diff
@@ -3,11 +3,18 @@
...
+from django.contrib.flatpages.admin import FlatPageAdmin
+from django.contrib.flatpages.models import FlatPage
+from django.db import models
+
 from geonode.people.admin import ProfileAdmin
 from geonode.people.forms import ProfileCreationForm
 from geonode.people.models import Profile
 from ordered_model.admin import OrderedModelAdmin
 
+from ckeditor.widgets import CKEditorWidget
```
Comment (dimasciput) [inline bims/admin.py]: Where did you get this from?

Comment (sonlinux) [reply bims/admin.py]: I have updated setup requirements.
It is from django-ckeditor package @dimasciput

File: core/urls.py [hunk 2]
```diff
@@ -26,6 +26,7 @@
     url(r'^grappelli/', include('grappelli.urls')),
     url(r'^accounts/', include('allauth.urls')),
     url('^contact/', include('contactus.urls')),
```
Comment (dimasciput) [inline core/urls.py]: Can you also add `r^` in contact?

**Comment by sonlinux** [comment]
Done

**Pull request merged**